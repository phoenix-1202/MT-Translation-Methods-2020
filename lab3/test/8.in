funWithIncorrectArgName :: Int -> Int
funWithIncorrectArgName a = a * b