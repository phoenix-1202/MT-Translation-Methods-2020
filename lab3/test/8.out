Incorrect argument b, expected name from arguments